src/background/certificate/helpers/CertificateErrorAnalyzer.test.ts

```diff
@@ -20,13 +20,21 @@ afterEach(() => {
 });
 
 test("returns CertificateError if a handled error code is provided", () => {
-  const requestDetails = { frameId: 0, error: "Error code 2153390067" };
+  const requestDetails = {
+    url: "",

```

Comment:
Where does this url come from? Why is it suddenly needed?

Reply:
The url is needed to handle the special case for the "about:blank" url.